### PR TITLE
style(list): print a warning if environment is empty

### LINKED ISSUE #930: flox list on empty environment prints an error
# Acceptance Criteria

**Given** I just ran `flox init` and have no packages installed
**When** I run `flox list`
**Then** I see a message, "No packages are installed."

### PR DESCRIPTION
**Pull request by ysndr**
fixes #930

### CODE REVIEW

**Comment by ghudgins** [comment]
looks good to me on the branch

**Comment by ysndr** [comment]
@ghudgins i figured this can also address #887, i extended the warning with a hint to `flox list --config`

**Comment by ghudgins** [comment]
great idea. agreed

File: cli/flox/src/commands/environment.rs [hunk 1]
```diff
@@ -945,51 +945,58 @@ impl List {
...
+            return Ok(());
+        }
+
+        let system = &flox.system;
+        let lockfile = Self::get_lockfile(&flox, &mut *env)?;
+        let packages = lockfile.list_packages(system);
+
+        if packages.is_empty() {
+            let message = formatdoc! {"
+                No packages are installed for tour current system ('{system}').
```
Comment (mkenigs) [inline cli/flox/src/commands/environment.rs:959]:
```suggestion
                No packages are installed for your current system ('{system}').
```

**Review by mkenigs — CHANGES_REQUESTED** [review]

**Review by mkenigs — APPROVED** [review]
nonblocking: add a test

**Pull request merged**